app/views/api/messages/_message.xml.builder

```diff
@@ -0,0 +1,12 @@
+xml.message do
+  xml.id message.id
+  xml.from_user_id message.from_user_id
+  xml.to_user_id message.to_user_id
+  xml.title message.title
+  xml.sent_on message.sent_on
+  xml.message_read message.message_read
+  xml.from_user_visible message.from_user_visible
+  xml.to_user_visible message.to_user_visible
+  xml.body_format message.body_format
+  xml.body message.body unless @skip_body
+end
```

Comment:
This is "notes api style" (ids as elements rather than attributes etc) which is an exception among osm apis.

Reply:
True, I modeled after notes api, without knowing that it is an exception. Will modify the spec to use attributes, with exception of `<body>` and `<title>`.